File: 009.py
def sol1():
    x = 0
    y = 0
    z = 0

    while x < 1001:
        y = x + 1
        while y < 1001:
            z = 1000 - x - y
            if -1 < z != y != x and x + y + z == 1000 and x**2 + y**2 == z**2:
                print(x)
                print(y)
                print(z)
                break
            y += 1
        x += 1

def sol2():
    for a in range(3, 1000):
        for b in range(a, 1000):
            c = (a*a+b*b)**0.5
            if c%1==0 and a+b+c==1000:
                print(int(a*b*c))

sol2()

File: 009.js
function perfSquareAndAddsUp(a, b, c, sum) {
    return c % 1 === 0 && a + b + c == 1000;
}

function sol1() {
    for (let a = 1; a < 1000; a++) {
        for (let b = a; b < 1000; b++) {
            c = Math.sqrt(a*a + b*b)
            if (c % 1 == 0 && a + b + c == 1000) {
                return a*b*c
            }
        }
    }
    return (0,0,0)
}

console.log(sol1())
